LICENSE-docs

```diff
@@ -0,0 +1,390 @@
+SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this a mistake?

Reply:
Should this crative commons

Reply:
@csantanapr yes good catch. It should be in the main `LICENSE` file. Fixed.